pages/disability/va-claim-exam.md

```diff
@@ -32,7 +32,9 @@ After you file your disability benefits claim, we may ask you to have a claim ex
 <div class="usa-alert usa-alert-warning">
 <div class="usa-alert-body">
 
-##### Be sure not to miss your scheduled exam.
+<h3 class="usa-alert-heading">
+Be sure not to miss your scheduled exam.
+</h3>
 
 If you miss your exam, we may not be able to reschedule you right away, and your claim may be delayed while you wait for a new appointment. Or we may need to rate your claim “as-is.” This means we’ll base our decision about your disability benefits only on the evidence we have in your file, which may not be enough.
 
```

Comment:
I think the text here should also be wrapped in a `.usa-alert-text`

Reply:
@ncksllvn - Interesting, I was going off of this sample text in the style guide that was linked in the issue:

https://department-of-veterans-affairs.github.io/vets-design-system-documentation/design-system/components/alertboxes.html

```html
<div class="usa-alert usa-alert-warning">
  <div class="usa-alert-body">
    <h3 class="usa-alert-heading">Warning status</h3>
    <p>Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
  </div>
</div>
```

I just tested adding that class locally, and there are no resulting styling changes upon inspection. 

Would you like me to add it?

Reply:
Ah yeah, we've been trying to standardize that HTML based on the US [Design System](https://designsystem.digital.gov/components/alerts/) and that looks like an outdated doc unfortunately. I don't think it has any visual effect - it's just easier to work with in the event we go through another visual overhaul to our alert style like we did recently. I only noticed it because I had the mentioned in one of my PRs the other day.

Would you mind adding it? Sorry, don't mean to be too nitpicky!